core/lib/user_lib.js

```diff
@@ -309,6 +310,31 @@ function _userGraph(userId) {
   });
 } // <= _userGraph
 
+/**
+ * @param {string} id
+ * @param {ProfilPatch} userPatch
+ * @return {Promise<UserDocument>}
+ */
+function updateProfil(id, userPatch) {
```

Comment:
At this time, the model is already validated. After investigation on previous code, we do not need to check other things.